insights/specs/__init__.py

```diff
@@ -127,6 +127,8 @@ class Specs(SpecSet):
     date = RegistryPoint(no_obfuscate=['hostname', 'ip'])
     date_utc = RegistryPoint(no_obfuscate=['hostname', 'ip'])
     db2ls_a_c = RegistryPoint(no_obfuscate=['hostname', 'ip'])
+    db2_database_configuration = RegistryPoint(multi_output=True, no_obfuscate=['hostname', 'ip'])
+    db2_database_manager = RegistryPoint(multi_output=True, no_obfuscate=['hostname', 'ip'])
```

Comment:
The 2 new specs should be in the front of the `db2ls_a_c` per the alphabet order.

Reply:
@xiangce - updated, please check